dist.ini

```diff
@@ -8,18 +8,21 @@ main_module = lib/Paws.pm
 [VersionFromMainModule]
 
 [Git::GatherDir]
-exclude_match=old_t
-exclude_match=TODO
-exclude_match=TODO_in_branch
-exclude_match=templates
-exclude_match=auto-lib
-exclude_match=builder-lib
-exclude_match=builder-bin
-exclude_match=botocore
-exclude_match=Makefile
+exclude_match    = ^old_t
+exclude_filename = TODO
+exclude_filename = TODO_in_branch
+exclude_match    = ^templates
+exclude_match    = ^auto-lib
+exclude_match    = ^builder-lib
+exclude_match    = ^builder-bin
+exclude_filename = botocore
```

Comment:
This should probably also be a match as botocore is a submodule

Reply:
can update it to be a match but as it stands the MANIFEST is the same

Reply:
both works pushing the update with a match for botocore